### ISSUE TITLE
current documentation describing units is not robust/consistent

### ISSUE THREAD

**Issue by celiotine** [COSMIC-PopSynth/COSMIC#224]

**Comment by scottcoughlin2014** [comment]
All of these things needs units defined
```
bpp_columns = ['tphys', 'mass_1', 'mass_2', 'kstar_1', 'kstar_2' ,
               'sep', 'porb', 'ecc', 'RROL_1', 'RROL_2', 'evol_type',
               'Vsys_1', 'Vsys_2', 'SNkick', 'SNtheta',
               'aj_1', 'aj_2', 'tms_1', 'tms_2',
               'massc_1', 'massc_2', 'rad_1', 'rad_2',
               'bin_num']
```
```
bcm_columns = ['tphys', 'kstar_1', 'mass0_1', 'mass_1', 'lumin_1', 'rad_1',
               'teff_1', 'massc_1', 'radc_1', 'menv_1', 'renv_1', 'epoch_1',
               'ospin_1', 'deltam_1', 'RROL_1', 'kstar_2', 'mass0_2', 'mass_2',
               'lumin_2', 'rad_2', 'teff_2', 'massc_2', 'radc_2', 'menv_2',
               'renv_2', 'epoch_2', 'ospin_2', 'deltam_2', 'RROL_2',
               'porb', 'sep', 'ecc', 'B_0_1', 'B_0_2',
               'SNkick_1', 'SNkick_2', 'Vsys_final', 'SNtheta_final',
               'SN_1', 'SN_2', 'bin_state', 'merger_type', 'bin_num']
```

**Comment by michaelzevin** [comment]
bpp_columns
- [ ] tphys
- [ ] mass_1
- [ ] mass_2
- [ ] kstar_1
- [ ] kstar_2
- [ ] sep
- [ ] porb
- [ ] ecc
- [ ] RROL_1
- [ ] RROL_2
- [ ] evol_type
- [ ] Vsys_1
- [ ] Vsys_2
- [ ] SNkick
- [ ] SNtheta
- [ ] aj_1
- [ ] aj_2
- [ ] tms_1
- [ ] tms_2
- [ ] massc_1
- [ ] massc_2
- [ ] rad_1
- [ ] rad_2
- [ ] bin_num

**Comment by michaelzevin** [comment]
bcm_columns

- [ ] tphys
- [ ] kstar_1
- [ ] mass0_1
- [ ] mass_1
- [ ] lumin_1
- [ ] rad_1
- [ ] teff_1
- [ ] massc_1
- [ ] radc_1
- [ ] menv_1
- [ ] renv_1
- [ ] epoch_1
- [ ] ospin_1
- [ ] deltam_1
- [ ] RROL_1
- [ ] kstar_2
- [ ] mass0_2
- [ ] mass_2
- [ ] lumin_2
- [ ] rad_2
- [ ] teff_2
- [ ] massc_2
- [ ] radc_2
- [ ] menv_2
- [ ] renv_2
- [ ] epoch_2
- [ ] ospin_2
- [ ] deltam_2
- [ ] RROL_2
- [ ] porb
- [ ] sep
- [ ] ecc
- [ ] B_0_1
- [ ] B_0_2
- [ ] SNkick_1
- [ ] SNkick_2
- [ ] Vsys_final
- [ ] SNtheta_final
- [ ] SN_1
- [ ] SN_2
- [ ] bin_state
- [ ] merger_type
- [ ] bin_num

**Comment by michaelzevin** [comment]
Closing this since it is addressed in #230
